pkg/api/app_miner.go

```diff
@@ -19,18 +19,18 @@ type MinerInfo struct {
 	Scheduler Scheduler
 }
 
-func (a *App) Miner() (*MinerInfo, error) {
+func (a *App) Miner() (MinerInfo, error) {
```

Comment:
this function is not going to return an error. There is always `nil`. Should be `func (a *App) Miner() MinerInfo`